cmake/DynamicVersion.cmake

```diff
@@ -112,6 +115,9 @@ function(dynamic_version)
 	`${OUTPUT_FOLDER}/.git_commit`
 	  Current commit
 
+	`${OUTPUT_FOLDER}/.git_distance`
+	  Current git distance from tag
+
```

Comment:
I don't think we need it in output file (for that matter `.git_describe`). The idea of those files is to be used as dependencies, e.g. for `configure_file` on a `config.h.in`. These always change at each commit.

Reply:
Ok, removed in latest commit. Since it's convenient to collect the short sha through `git describe`, I went ahead and stored it in json. But I didn't write it to file either.